src/main/java/com/jherrick/SubBoard.java

```diff
@@ -122,7 +122,7 @@ public boolean isWin() {
 	 * NOTE: isWon() must always be called before isDrawn()
 	 **/
 	public boolean isDraw() {
-		if (~(xBoard | oBoard) == 0) {
+		if (~(xBoard | oBoard) == 511) {
```

Comment:
Why does this calculation work? I'd think that it would either be 

xBoard | oBoard == 511 (aka 111111111)

or 

~(xBoard | oBoard) == 0

The one's complement of 111111111 is 000000000.

Reply:
Added a new commit removing the ~

New test within the if statement should be (XBoard | OBoard) == 511